README.md

```diff
@@ -1,3 +1,4 @@
+**Notice:** This repository is for the **old Arduino IDE** which is no longer in active development. For the latest version, please visit the [new Arduino IDE repository](https://github.com/arduino/Arduino-ide). This helps in maintaining focus and directing efforts where they are most needed.
```

Comment:
```suggestion
**Notice:** This repository is for **Arduino IDE 1.x** which is no longer in active development. For the latest version, please visit the [new Arduino IDE repository](https://github.com/arduino/arduino-ide).

```

- Replace "old Arduino IDE" with "Arduino IDE 1.x". "Old Arduino IDE" is ambiguous.
- Insignificant case correction in URL
- Remove superfluous sentence